source/direct_debit/index.html.md.erb

```diff
@@ -0,0 +1,322 @@
+
+The same API call is used for single payments in all contexts, including
+Direct Debit.
+
+You cannot take further payments from users using this method without creating
+a new payment. This requires users to re-enter their account details.
+
+## Variable payments using Direct Debit
+
+A variable agreement allows you to collect payments of any amount at any time.
+For example you may want to collect:
+
+* money each time a service is provided
+
+* payments where the amounts vary according to service usage
+
+* regular payments where you do not know in advance how much each payment will be
+
+You could also use the variable functionality to simulate a recurring or
+scheduled Direct Debit payment, by sending an API call on a set schedule.
+Users would be emailed each time a payment is collected against their mandate.
+
+### Variable Direct Debit payment process
+
+These steps happen in order.
+
+#### 1. Agreement is created
+
+API calL:  `POST /v1/agreements/`
+
+Your service must specify:
+
+* a `return_url` where GOV.UK Pay will direct the user when they have finished setting up the agreement
+
+* the `agreement_type` (`on demand`)
+
+* a `reference` (optional) - your reference for this agreement
+
+The response contains a `next_url`.
+
+#### 2. Your service directs the user to the `next_url`
+
+#### 3. The user provides their account details
+
+For example, account number and sort code for Direct Debit
+
+GOV.UK Pay makes an API call to our Direct Debit supplier to verify account
+details.
+
+#### 4. GOV.UK Pay presents the user with a confirmation screen
+
+The screen presents the user's account details. The user grants their consent
+by selecting __Confirm__.  GOV.UK Pay sends an email to the user to confirm
+that their mandate has been set up. If the mandate subsequently fails, the
+GOV.UK Pay team will send an email to update the user that their mandate could
+not be set up.
+
+#### 5. GOV.UK Pay securely passes the user's details on to the Direct Debit provider
+
+The Direct Debit provider submits a request to BACS to set up a mandate, and
+sends a mandate ID to GOV.UK Pay. At this point, services can send a payment
+request.
+
+#### 6. GOV.UK Pay returns the user to your service using the `return_url`
+
+#### 7. Call the API to check the status of the agreement
+
+API call: `GET /v1/agreements/<AGREEMENT-ID>`
+
+This is to display an appropriate screen to the user. For a Direct Debit
+payment, this will initially be in a ‘pending’ state until the bank confirms
+the mandate. The mandate confirmation process takes 2 working days.
+
+#### 8. The GOV.UK Pay admin tool updates the transaction status
+
+When GOV.UK Pay receives confirmation that the mandate is
+confirmed, the GOV.UK Pay admin tool will update the transaction status.
+
+### Collecting payments against a mandate
+
+API call: `POST /v1/payments`
+
+Required arguments:
+
+```
+{
+ "amount": 2000,
+ "description": "fml",
```

Comment:
Should we have a description key with the value of fml? might be interpreted as something rude

Reply:
Oh my goodness